doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -205,20 +230,43 @@ certain position of the robot as a **Home** position.
   how poses are associated with particular groups. You can save
   individual poses for each group.
 
+* Select the *panda_arm* and define a **ready** pose for it with the following joint values {0.0, -0.785, 0.0, -2.356, 0.0, 1.571, 0.785}.
```

Comment:
Use code font (double backticks) for the joint values

Reply:
Fixed it. Thanks for pointing this out!